src/loader/AMASSLoader.py

```diff
@@ -0,0 +1,320 @@
+
+    As for all loaders it is possible to add custom properties to the loaded object, for that use add_properties.
+
+    Finally it sets all objects to have a category_id corresponding to the void class,
+    so it wouldn't trigger an exception in the SegMapRenderer.
+
+    Note: if this module is used with another loader that loads objects with semantic mapping, make sure the other
+    module is loaded first in the config file.
+
+    Example 1: generate a pose of human kicking a ball (male model). by looking into different categories recorded in mocap datasets supported by AMASS database, we can see that in CMU dataset, category number 10 contains multiple trials of human kicking a ball.
+
+    {
+        "module": "loader.AMASSLoader",
+        "config": {
+        "data_path": "<args:0>",
+        "sub_dataset_id": "CMU",
+        "body_model_gender": "male",
+        "subject_id": "10",
+        "sequence_id": "1",
+        "frame_id": "600",
+        "cf_set_shading": "SMOOTH"
+      },
+    }
+
+    Example 2: generate a pose of human picking up golf ball (female model). from CMU dataset, choose subject number 64 which contains multiple trials of human playing golf, when frame_id is not specified, random frame will be chosen from the motion trial
+
+    {
+        "module": "loader.AMASSLoader",
+        "config": {
+        "data_path": "<args:0>",
+        "sub_dataset_id": "CMU",
+        "body_model_gender": "female",
+        "subject_id": "64",
+        "sequence_id": "27",
+        "frame_id": "",
+        "cf_set_shading": "SMOOTH"
+      },
+    }
+
+    **Configuration**:
+
+    .. csv-table::
+       :header: "Parameter", "Description"
+
+       "data_path", "The path to the AMASS Dataset folder in resources folder. Type: string. Default: 'resources/AMASS'."
+       "_used_sub_dataset_id", "Identifier for the sub dataset, the dataset which the human pose object should be extracted from. Type: srtring." 
+       "_used_body_model_gender", "The model gender, pose will represented using male, female or neutral body shape. Type: string." 
+       "_used_subject_id", "Type of motion from which the pose should be extracted, this is dataset dependent parameter. Type: int." 
+       "_used_sequence_id", "Sequence id in the dataset, sequences are the motion recorded to represent certain action. Type: int." 
+       "_used_frame_id", "Frame id in a selected motion sequence. Type: int."
+    """
+
+    # list of all possible supported mocap datasets: ['CMU', 'Transitions_mocap', 'MPI_Limits', 'SSM_synced',
+    # 'TotalCapture', 'Eyes_Japan_Dataset', 'MPI_mosh', 'MPI_HDM05', 'HumanEva', 'ACCAD', 'EKUT', 'SFU', 'KIT',
+    # 'H36M', 'TCD_handMocap', 'BML']
```

Comment:
This should be denoted as Available values for the `sub_dataset_id`:

From the `CONTRIBUTION.md`:

> "param_a", "Used for this/means this. Type: type. Default: value. Available: [some_value, value, another_value]."

Reply:
fixed